defaults/main.yml

```
@@ -73,6 +73,7 @@ oauth_consul_service_name: 'oauth-proxy'
 oauth_consul_service_id: 'oauth-proxy:{{ oauth_domain }}'
 oauth_consul_service_tags: ['oauth-proxy', 'ssl-proxy-backend', 'site']
 
+oauth_nested_compose: false
```

Maybe a more descriptive name like:
```suggestion
oauth_compose_no_start: false
```